src/azure-cli/azure/cli/command_modules/acr/connected_registry.py

```diff
@@ -421,25 +413,19 @@ def _get_install_info(cmd,
             password1=True, password2=True, resource_group_name=resource_group_name)
         credentials = LongRunningOperation(cmd.cli_ctx)(poller)
         sync_username = credentials.username
-        sync_password = {
-            "password1": credentials.passwords[0].value,
-            "password2": credentials.passwords[1].value
-        }
+        sync_password = credentials.passwords[0].value
         logger.warning('Please store your generated credentials safely.')
     else:
         sync_username = sync_token_name
         sync_password = "<sync token password>"
 
     logger.warning("Value 'ACR_SYNC_TOKEN_USERNAME' has been deprecated and will be removed in a future release."
```

Comment:
@xyxyxyxyxyxy do we still want to expose this warning message if we no longer use env vars?

Reply:
@xyxyxyxyxyxy  can you modify the warning to say that all the env variables are going to be deprecated?

Reply:
We keep these env vars for now, and I have changed the warning.